components/Layout.js

```diff
@@ -31,7 +31,7 @@ const Layout = ({ links, children, title, description }) => {
   const [secondaryNavbar, setSecondaryNavbar] = useState(false);
 
   useEffect(() => {
-    if (currentPath === "/hackrplay/2022/home") {
+    if (currentPath === "/hackrplay/2022/home" || currentPath === "/2playsamonth/2023/home") {
```

Comment:
Please change it to 2023/2playsamonth/home

Reply:
We will go with the activity name at the top.